drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -682,11 +668,32 @@ def _get_item_state(self):
         # handler and manipulated there to hide correctness of items placed
```

Comment:
@mtyaka "... and *the data it returns is* manipulated there ..."?

(Again, this is an existing comment, but since there's a lot going on in this code I think it's worth being precise in any comments that explain what's going on.)